tests/performance/test_performance.py

```diff
@@ -22,11 +24,13 @@ def get_instance(obj, **kwargs):
     return instance
 
 
-BASE = os.path.dirname(__file__)
-TESTS = glob.glob(BASE + '/test_cases/*.json')
+TEST_CASES_PATH = pathlib.Path(__file__).parent / 'test_cases'
+TEST_CASES_PATH_LEN = len(str(TEST_CASES_PATH)) + 1
+TEST_CASES = [str(test_case) for test_case in TEST_CASES_PATH.rglob('*.json')]
+IDS = [test_case[TEST_CASES_PATH_LEN:] for test_case in TEST_CASES]
```

Comment:
what are the ids for?

Reply:
A very minor thing actually: when running the test using `pytest -v` you get one line per test case, with its id:

```
$ pytest -v --disable-warnings tests/performance/test_performance.py 
========================================================= test session starts =========================================================
platform linux -- Python 3.8.10, pytest-6.2.4, py-1.10.0, pluggy-0.13.1 -- /mnt/nvme0n1p2/xals/.virtualenvs/RDT.clean/bin/python
cachedir: .pytest_cache
rootdir: /home/xals/Projects/MIT/RDT.clean, configfile: setup.cfg
plugins: cov-2.12.1
collected 4 items                                                                                                                     

tests/performance/test_performance.py::test_performance[numerical/NumericalTransformer/RandomInteger1000.json] PASSED           [ 25%]
tests/performance/test_performance.py::test_performance[numerical/NumericalTransformer/RandomIntegerNaNs1000.json] PASSED       [ 50%]
tests/performance/test_performance.py::test_performance[numerical/NumericalTransformer/RandomIntegerNaNs10000.json] PASSED      [ 75%]
tests/performance/test_performance.py::test_performance[numerical/NumericalTransformer/RandomInteger10000.json] PASSED          [100%]

==================================================== 4 passed, 5 warnings in 0.94s ====================================================
```